lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Another worry is that the `File.open` check will actually fail for bare git repositories, i.e. repositories that don't have a working directory (but that might still contain the video files).